pix-editor/app/styles/_challenges.scss

```
@@ -207,6 +207,10 @@ img.clickable {
       border-radius: 0.28571429rem;
     }
 
+    .ProseMirror {
+      font-family: Lato, 'Helvetica Neue', Arial, Helvetica, sans-serif;
```

Ça ne respecte pas les typographies du Design System https://ui.pix.fr/?path=/docs/utiliser-pix-ui-design-tokens-typographie--docs

---

Je vous recommande plutôt :

```suggestion
      @extend %pix-body-m;
```

Qui tirera tout ce dont vous avez besoin.

Si c'est pas ok dans ce contexte il faudrait voir avec le design ce qu'on veut utiliser !

---

L'intérêt est que si les fonts évoluent (choix design), il n'y a pas de reprise de code nécessaire ;)

---

on peut essayer de voir si ça chamboule pas trop tout !